ArduPlane/mode_takeoff.cpp

```diff
@@ -65,7 +65,7 @@ void ModeTakeoff::update()
 {
     if (!takeoff_started) {
         // see if we will skip takeoff as already flying
-        if (plane.is_flying() && plane.ahrs.groundspeed() > 3) {
+        if ((millis() - plane.started_flying_ms > 10000U) && plane.ahrs.groundspeed() > 3) {
```

Comment:
This doesn't look correct.  You do need to check `is_flying()` as well.

Reply:
added....thought it was reset if not flying...but appears to reset only if flying is restarted